### PR TITLE
[release-1.26] Allow restricted pod security for upgrade tests

### PR DESCRIPTION
**Pull request by mgencur**
Bring in changes from https://issues.redhat.com/browse/SRVKE-1331

## Proposed Changes

- Vendor upstream/midstream changes for running with restricted pod security standard
-
-

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -169,10 +169,10 @@ require (
...
 replace (
 	// Knative forks.
-	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20220926135101-4a89adaca7d1
-	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20220504141528-c59955f23883
-	knative.dev/eventing-kafka-broker => github.com/openshift-knative/eventing-kafka-broker v0.25.1-0.20220927171706-e0d47343c338
-	knative.dev/serving => github.com/openshift/knative-serving v0.10.1-0.20220926120634-72a6f7ea6cfb
+	knative.dev/eventing => github.com/openshift/knative-eventing v0.99.1-0.20221109093310-866f17f1f1f9
+	knative.dev/eventing-kafka => github.com/openshift-knative/eventing-kafka v0.19.1-0.20221109100552-26eea5293bca
+	knative.dev/eventing-kafka-broker => github.com/openshift-knative/eventing-kafka-broker v0.25.1-0.20221109150735-a948d166b5ee
+	knative.dev/serving => github.com/openshift/knative-serving v0.10.1-0.20221104105313-b02fdc605c48
```
Comment (skonto) [inline go.mod:175]: Cool as discussed in slack we should have latest.
Serving last commit on 1.5 is `b02fdc605c4811b418dcb7e53547e17d4d688631` so seems right to me.

**Comment by mgencur** [comment]
Looks like the upgrade tests on 4.11 just timed out after 1hour. I don't see any other error.

**Comment by mgencur** [comment]
There are a lot of duplicated events and gathering traces for all the events makes the test suite timeout. There's a limit of 1000 traces for each test which might be too much. Let's discuss changing this.

**Comment by mgencur** [comment]
The latest commit pulls in changes from https://github.com/openshift/knative-eventing/pull/1923 to prevent the test suite timeout.

**Comment by mgencur** [comment]
It's the known KafkaSinkSourceContinualTest, see past search for this failure: https://search.ci.openshift.org/?search=KafkaSinkSourceContinualTest&maxAge=336h&context=2&type=bug%2Bjunit&name=%5Eperiodic-ci.*openshift-knative-serverless-operator.*&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job

**Comment by mgencur** [comment]
I still see a few events like this one:
```
"message": "0/8 nodes are available: 1 node(s) had untolerated taint {zipkin: }, 3 node(s) had untolerated taint {node-role.kubernetes.io/master: }, 4 Insufficient cpu. preemption: 0/8 nodes are available: 4 No preemption victims found for incoming pod, 4 Preemption is not helpful for scheduling.",
```
[Link to events](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-knative_serverless-operator/1804/pull-ci-openshift-knative-serverless-operator-release-1.26-4.11-upgrade-tests-aws-ocp-411/1592099127125086208/artifacts/upgrade-tests-aws-ocp-411/gather-extra/artifacts/events.json)

**Pull request merged**